Project.toml

```
@@ -1,7 +1,7 @@
 name = "Wavelets"
 uuid = "29a6e085-ba6d-5f35-a997-948ac2efa89a"
 author = ["Gudmundur Adalsteinsson"]
-version = "0.9.3"
+version = "0.9.4"
```

```suggestion
version = "0.9.3"
```

0.9.3 is not released yet: https://github.com/JuliaRegistries/General/blob/3626f1994b11a7872499e6f696f0ca0ed3c96db3/W/Wavelets/Versions.toml

---

oh, i saw 0.9.3 in the master branch file so i figured i needed to go one beyond that.  i'll undo.